lib/rules/eol-last.js

```diff
@@ -5,6 +5,10 @@ import Rule from './_base.js';
 
 export default class EolLast extends Rule {
   parseConfig(config) {
+    if (!this.isStrictMode) {
```

Comment:
Can you add a comment here explaining why the rule is off here and what strict mode means?